src/runtime/config/configuration-qemu-sev-snp.toml.in

```diff
@@ -0,0 +1,664 @@
+# Copyright 2022 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# XXX: WARNING: this file is auto-generated.
+# XXX:
+# XXX: Source file: "@CONFIG_QEMU_IN@"
+# XXX: Project:
+# XXX:   Name: @PROJECT_NAME@
+# XXX:   Type: @PROJECT_TYPE@
+
+# Basing SNP updates on this post from Joana
+# https://github.com/kata-containers/kata-containers/blob/main/docs/how-to/how-to-run-kata-containers-with-SNP-VMs.md
+
+[hypervisor.qemu]
+path = "@QEMUPATH@"
+kernel = "@KERNELSEVPATH@"
+#SNP should be able to use SEV kernel and initrd with things being passed via kernel params
+initrd = "@INITRDSEVPATH@"
+machine_type = "@MACHINETYPE@"
+
+# Enable confidential guest support.
+# Toggling that setting may trigger different hardware features, ranging
+# from memory encryption to both memory and CPU-state encryption and integrity.
+# The Kata Containers runtime dynamically detects the available feature set and
+# aims at enabling the largest possible one, returning an error if none is
+# available, or none is supported by the hypervisor.
+#
+# Known limitations:
+# * Does not work by design:
+#   - CPU Hotplug 
+#   - Memory Hotplug
+#   - NVDIMM devices
+#
+# Default false
+confidential_guest = true
+
+# enable SEV SNP VMs
+sev_snp_guest = @DEFSNPGUEST@
+
+# SNP guest policy
+# From Niteesh's https://github.com/kata-containers/kata-containers/pull/5796/files
+# (this will depend on that patch)
+snp_guest_policy=@DEFSNPGUESTPOLICY@
+
+
+# Enable running QEMU VMM as a non-root user.
+# By default QEMU VMM run as root. When this is set to true, QEMU VMM process runs as
+# a non-root random user. See documentation for the limitations of this mode.
+# rootless = true
+
+# List of valid annotation names for the hypervisor
+# Each member of the list is a regular expression, which is the base name
+# of the annotation, e.g. "path" for io.katacontainers.config.hypervisor.path"
+enable_annotations = @DEFENABLEANNOTATIONS@
+
+# List of valid annotations values for the hypervisor
+# Each member of the list is a path pattern as described by glob(3).
+# The default if not set is empty (all annotations rejected.)
+# Your distribution recommends: @QEMUVALIDHYPERVISORPATHS@
+valid_hypervisor_paths = @QEMUVALIDHYPERVISORPATHS@
```

Comment:
```suggestion
valid_hypervisor_paths = @QEMUSNPVALIDHYPERVISORPATHS@
```
if you align with https://github.com/kata-containers/kata-containers/pull/6062#issuecomment-1406581382